File: 프로그래머스/lv1/82612. 부족한 금액 계산하기/부족한 금액 계산하기.js
function solution(price, money, count) {
    var answer = 0;
    for (let i = 1; i <= count; i++) {
        answer = answer + price*i;
    }
        
    (money > answer) ? answer = 0 : answer = (answer - money);

    return answer;
}

console.log(solution(3, 20, 4));

File: 프로그래머스/1/82612. 부족한 금액 계산하기/부족한 금액 계산하기.java
class Solution {
    public long solution(int price, int money, int count) {
        long cost = 0;
        
        for (int i=0; i<count+1; i++) {
            cost += price*i;
        }
        
        if (money < cost) return cost-money;
        else return 0;
            
    }
}